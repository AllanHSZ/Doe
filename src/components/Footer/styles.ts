import styled, { css } from 'styled-components'

export const Footer = styled.footer`
  ${({ theme }) => css`
    color: ${theme.colors.text};
    font-size: ${theme.font.sizes.xsmall};
  `}
`
