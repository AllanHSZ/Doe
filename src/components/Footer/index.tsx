import * as S from './styles'

const Footer = () => (
  <S.Footer>
    <S.FooterText>Doe Â© 2021, Nenhum direito reservado.</S.FooterText>
  </S.Footer>
)

export default Footer
