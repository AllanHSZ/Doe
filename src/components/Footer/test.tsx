import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import Footer from '.'

describe('<Footer />', () => {
  it('should render correctly', () => {
    const { container } = renderWithTheme(<Footer />)

    expect(
      screen.getByText(/Doe Â© 2021, Nenhum direito reservado/i)
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
