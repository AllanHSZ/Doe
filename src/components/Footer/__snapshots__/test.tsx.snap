// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
.c0 {
  color: #394338;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
}

.c1 {
  font-size: 1.2rem;
}

<footer
  class="c0"
>
  <p
    class="c1"
  >
    Doe Â© 2021, Nenhum direito reservado.
  </p>
</footer>
`
